acceptance/tests/ssl/crl_retrieval_and_replacement.rb

```diff
@@ -0,0 +1,48 @@
+test_name "Crl retrieval and replacement from master" do
+
+  tag 'risk:medium',
+      'server'
+
+  with_puppet_running_on(master, {}) do
+    agents.each do |agent|
+      ca_crl_path = on(master, puppet('config print cacrl --section master')).stdout
+      crl_path = on(agent, puppet('config print hostcrl --section agent')).stdout
```

Comment:
for these you can use `puppet_config` (I just discovered and used it [here](https://github.com/puppetlabs/puppetserver/pull/1733/files#diff-1a16d5f5705343389c819fc4670b1aeeR111))